etc/dev/start

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
How did you create the alias? Showing up here as an entirely new file.

Reply:
I used the `ln` command, but I forgot to use the `-s` flag to make sure it creates a symlink as opposed to a hardlink. 

I removed and re-created the alias with the `-s` flag, and this appears to be fixed now.